kythe/typescript/indexer.ts

```diff
@@ -1249,7 +1249,7 @@ class Visitor {
    * and that case is handled as part of the ordinary declaration handling.
    */
   visitExportDeclaration(decl: ts.ExportDeclaration) {
-    if (decl.exportClause) {
+    if (decl.exportClause && ts.isNamedExports(decl.exportClause)) {
```

Comment:
Can you change the package.json / lockfile too?

Reply:
Or is this function in ts3.7

Reply:
Updated!